packages/firebase_ml_vision/lib/src/barcode_detector.dart

```diff
@@ -295,6 +296,11 @@ class Barcode {
   /// Could be null if the bounding rectangle can not be determined.
   final Rect boundingBox;
 
+  /// Barcode bytes as they were encoded in the barcode.
+  ///
+  /// Null if nothing found.
+  final Uint8List rawBytes;
```

Comment:
I am not really fond of having the native side transmit the bytes list without the client requesting for it.
That'll make the communication slower for everyone, even those who aren't using `rawBytes`.

Thoughts @Salakar?

Reply:
@Salakar @rrousselGit Any more insight into this? :) How would you prefer to go about this alternatively?